tests/build/build.go

```diff
@@ -30,6 +30,7 @@ const (
 	helloWorldComponentGitSourceRepoName = "devfile-sample-hello-world"
 	pythonComponentGitSourceURL          = "https://github.com/redhat-appstudio-qe/devfile-sample-python-basic"
 	dummyPipelineBundleRef               = "quay.io/redhat-appstudio-qe/dummy-pipeline-bundle:latest"
+	buildTemplatesTestLabel              = "build-templates-e2e"
```

Comment:
label used by [build-definitions' e2e test](https://github.com/redhat-appstudio/build-definitions/blob/01f213fd04c098a5c8d4d1c5243c0656aa3cb4cc/.tekton/pull-request.yaml#L117)

Reply:
Thanks so much for making it!